feat(core): add NO_ERRORS_SCHEMA, which disabled all schema checks

vsavkin opened the pull request.

juliemr commented on modules/@angular/platform-browser/test/testing_public_spec.ts:
```diff
@@ -253,6 +253,7 @@ export function main() {
                expect(TestBed.createComponent(ComponentUsingInvalidProperty).componentInstance)
                    .toBeAnInstanceOf(ComponentUsingInvalidProperty);
              });
+
```
Revert

juliemr commented:
LGTM. It would be nice if the commit message had an example of why you would want to use this.

vsavkin committed:
feat(core): add NO_ERRORS_SCHEMA that allows any properties to be set on any element

Often it is useful to test a component without rendering certain directives/components
in its template because these directives require some complicated setup.

You can do that by using NO_ERRORS_SCHEMA.

TestBed.configureTestingModule({
  schemas: [NO_ERRORS_SCHEMA]
});

This would disable all schema checks in your tests.
